src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -860,6 +860,11 @@ export interface ITerminalInstance {
 	 * Activates the most recent link of the given type.
 	 */
 	openRecentLink(type: 'localFile' | 'url'): Promise<void>;
+
+	/**
+	 * @param data to write to the terminal
+	 */
+	write(data: string): void;
```

Comment:
Ideally we wouldn't expose this

Reply:
was trying to avoid adding an event because we have so many, but can do

Reply:
I actually am not sure what the alternative to this would be